contracts/Market.sol

```diff
@@ -175,17 +168,6 @@ contract Market is PausableAccessPreset {
     emit PriorityRestrictedThresholdSet(threshold);
   }
 
-  /**
-   * The amount of supply available for anyone to buy.
-   */
-  function totalUnrestrictedSupply() external view returns (uint256) {
-    uint256 activeSupply = _removal.cumulativeBalanceOf(address(this));
-    return
-      activeSupply < _priorityRestrictedThreshold
-        ? 0
-        : activeSupply - _priorityRestrictedThreshold; // todo compare this against trySub?
-  }
-
```

Comment:
i'm pretty sure we still want this function because restricted supply is not the same thing as reserved supply.  this has to do with priority restricted threshold that we hold for allowlisted buyers when supply gets low, totally different concept from the reserved removal pool

Reply:
it can be retrieved without a helper function

Reply:
oh good point, by getting the restriction threshold and the active supply we can calculate that oflfine